third_party/openmetadata/Makefile

```diff
@@ -6,8 +6,18 @@ deploy-openmetadata:
 	helm repo add open-metadata https://helm.open-metadata.org/ || true
 	helm repo update || true
 	kubectl create ns ${INSTALLATION_NAMESPACE} || true
-	sed s/INSTALLATION_NAMESPACE/${INSTALLATION_NAMESPACE}/g pv1.yaml | kubectl apply -n ${INSTALLATION_NAMESPACE} -f - || true
-	sed s/INSTALLATION_NAMESPACE/${INSTALLATION_NAMESPACE}/g pv2.yaml | kubectl apply -n ${INSTALLATION_NAMESPACE} -f - || true
+ifeq ($(OPENSHIFT_INSTALLATION), true)
```

Comment:
Maybe to call the sub directory `ibm-openshift`, we use IBM specific storage classes.

Reply:
done